src/openzaak/components/zaken/api/viewsets.py

```diff
@@ -868,7 +868,7 @@ def _get_zaak(self):
     def get_serializer_context(self):
         context = super().get_serializer_context()
         # DRF introspection
-        if not self.kwargs:
+        if not getattr(self, 'kwargs', None):
```

Comment:
`kwargs` did not seem to be an attribute of `self` so this threw the exception

```python
Traceback (most recent call last):
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/contrib/admin/options.py", line 606, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/utils/decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/views/decorators/cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/contrib/admin/sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/contrib/admin/options.py", line 1645, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/utils/decorators.py", line 45, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/utils/decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/contrib/admin/options.py", line 1529, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/django/contrib/admin/options.py", line 1572, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "/Users/shea/Projects/open-zaak/src/openzaak/utils/admin.py", line 213, in save_model
    data = self.get_serializer_data(request, viewset, obj)
  File "/Users/shea/Projects/open-zaak/src/openzaak/utils/admin.py", line 163, in get_serializer_data
    serializer = viewset.get_serializer(obj)
  File "/Users/shea/Projects/open-zaak/env/lib/python3.7/site-packages/rest_framework/generics.py", line 111, in get_serializer
    kwargs['context'] = self.get_serializer_context()
  File "/Users/shea/Projects/open-zaak/src/openzaak/components/zaken/api/viewsets.py", line 871, in get_serializer_context
    if not self.kwargs:
AttributeError: 'ZaakBesluitViewSet' object has no attribute 'kwargs'
```

Now if there is no attribute `kwargs` then `None` is just returned so we just return the `context` from this function rather than raising an exception.